demo/components/bitmovinConviva/ContentMetadataBuilder.brs

```diff
@@ -19,13 +19,14 @@ function build()
     if m.contentMetadata.assetName = invalid
       m.contentMetadata.assetName = getAssetName()
     end if
-
+    
     m.contentMetadata.viewerId = getViewerId()
     m.contentMetadata.isLive = getStreamType()
     m.contentMetadata.playerName = getApplicationName()
     m.contentMetadata.contentlength = getDuration()
-
-    m.contentMetadata.tags = getCustom()
+    m.contentMetadata.customMetadata = {}
+    m.contentMetadata.customMetadata.SetModeCaseSensitive()
```

Comment:
Can you quickly elaborate what `.SetModeCaseSensitive()` is about 🤔

Reply:
In Roku roAssociativeArrays doesn't respect case sensitivity. So here we force it to consider case sensitivity